pkgs/servers/mail/listmonk/default.nix

```diff
@@ -44,10 +36,10 @@ buildGoModule rec {
         "i18n:/i18n"
       ];
     in
-      ''
-        stuffbin -a stuff -in $out/bin/listmonk -out $out/bin/listmonk \
-          ${lib.concatStringsSep " " vfsMappings}
-      '';
+    ''
+      stuffbin -a stuff -in $out/bin/listmonk -out $out/bin/listmonk \
```

Comment:
Can you move the formatting changes in their own commit? This will make the blame/log cleaner.